test_mobile_wallet.sh

```
@@ -0,0 +1,138 @@
+
+# Test 3: Run code analysis
+print_status "Running code analysis..."
+melos analyze || print_error "Code analysis failed"
+
+# Test 4: Run unit tests
+print_status "Running unit tests..."
+melos test || print_error "Unit tests failed"
+
+# Test 5: Check if example app can be built
+print_status "Testing example app build..."
+cd examples/mobile_wallet_example
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)